.circleci/config.yml

```diff
@@ -129,19 +129,18 @@ workflows:
           matrix:
             parameters:
               # FIXME other things worth adding to the matrix:
-              # - clojurescript
               # - cider-nrepl
               # - node.js runtimes
-              clojure_version: ["1.8", "1.9", "1.10", "master"]
+              clojure_version: ["1.9", "1.10", "1.11", "master"]
```

Comment:
I guess it still works on Clojure 1.8, right? If not - a changelog entry is in order.